CHANGELOG.md

```diff
@@ -6,14 +6,16 @@
 
 ### Bug Fixes
 
+* [ObjectServer] Fixed a crash when an authentication error happend (#4726).
```

Comment:
The logger should only run the formatter code if that level is enabled, otherwise it will just ignore it. At least that was the intent.